qt/python/mantidqt/widgets/fitpropertybrowser/fitpropertybrowser.py

```diff
@@ -80,10 +113,17 @@ def _get_allowed_spectra(self):
         for ax in self.canvas.figure.get_axes():
             try:
                 for ws_name, artists in ax.tracked_workspaces.items():
+                    # we don't want to include the fit workspace in our selection
+                    # unfortunately, this might catch some workspaces which have _Workspace appended to their name
+                    # hopefully this is an unlikely scenario.
+                    if ws_name.endswith("_Workspace"):
```

Comment:
I think there's a more elegant way of doing this that I implemented in the EngDiff UI when overloading  `_get_allowed_spectra` (which I think it a natural place to put this) that will discriminate between output workspaces from the fit and any workspace ending with `_Workspace` 
https://github.com/mantidproject/mantid/blob/dec8b0f0596373b906024f13c882f6881016728e/scripts/Engineering/gui/engineering_diffraction/tabs/fitting/plotting/EngDiff_fitpropertybrowser.py#L107

Reply:
So `output_wsnames` will include all the `_Workspace`, `_Parameters `etc. workspaces - though I'm not sure they are updated if e.g. the user was to rename one of the output workspaces (i.e. I think whatever is returned by `self.getWorkspaceList()` is updated at the point of the fit completion, it is not a dynamically updated thing)

Reply:
Thanks for pointing me towards that - looks like a more robust approach